AutomaticQC/imosSpikeQC.txt

```diff
@@ -0,0 +1,30 @@
+% comma separated variable names/subnames
+allowed_variables = CDOM, CNDC, CPHL, OXY, PAR, PSAL, SST, TEMP, TURB
```

Comment:
why only this set of variables? Is there a reason we're not doing this on the whole set ?
https://github.com/aodn/imos-toolbox/blob/master/IMOS/imosParameters.txt

Reply:
yeap, fencing on purpose for several reasons:

1. I don't want people applying this to untested variables,  ATM.
2. I didn't code the cases for two or more dimensional variables (e.g. velocity).
3. I got a very polluted UI when there was a lot of variables because I had to add text + dropdown menu + buttons. This was dragging me down too. Hence, I fenced down the vars to the most used/ful ones and simplified the interface.

The idea is to expand this, but only after I managed a feasible UI design that allows users to quickly swap values/methods for different variables for example.